### PR TITLE
Adding tapes

### PR DESCRIPTION
**Pull request by vijetmahabaleshwar-okta**
* Adding scenario tapes
* Fixing a minor bug in the test server

### CODE REVIEW

**Comment by lboyette-okta** [comment]
@bretterer this requires a change to the way we record the tapes. I'll change it to localhost:DEFAULT_PORT/scenario_name, so when you get a response, if you use the link, it'll work

**Comment by nbarbettini** [comment]
Some of the names seem inconsistent - `list-users` vs `user-get`. (Sometime's it's prefixed by the resource type, and sometimes it's prefixed by the action)

Otherwise, LGTM.

File: scenarios/group-get-and-stats.har [hunk 1]
```diff
@@ -0,0 +1,263 @@
+{
```
Comment (lboyette-okta) [inline scenarios/group-get-and-stats.har]: `stats` doesn't exist anymore

Comment (vijetmahabaleshwar-okta) [inline scenarios/group-get-and-stats.har]: Ah... I have removed the call to group-stats but forgot to rename the file. The .har file should be fine.

**Comment by bdemers** [comment]
I just merged master into this branch

**Pull request merged**